webpack/ForemanTasks/Components/TasksTable/SubTasksPage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import TasksTablePage from './';
+import { getSubTaskID } from './TasksTableHelpers';
+
+export const SubTasksPage = props => {
+  const getBreadcrumbs = actionName => ({
+    breadcrumbItems: [
+      { caption: __('Tasks'), url: `/foreman_tasks/tasks` },
+      {
+        caption: actionName,
+        url: `/foreman_tasks/tasks/${getSubTaskID(window.location)}`,
```

Comment:
Instead `window.location`, can't we use the one provided by `react-router`?

Reply:
Chagned, thanks